docs/deploying-airbyte/deploying-airbyte.md

```diff
@@ -0,0 +1,182 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploying Airbyte
+
+The Airbyte platform is a sophisticated data integration platform that enables you to handle large amounts of data movement. If setting up an Airbyte server does not fit your usecase needs (i.e. you're using Jupyter Notebooks or iterating on an early prototype for your project) you may find the [PyAirbyte](./using-airbyte/pyairbyte/getting-started) documentation useful. To quickly deploy Airbyte on your local machine you can visit the 
```

Comment:
We'd rather people use the quickstart over pyairbyte. Can we change the order of sentences to emphasize the quickstart over pyairbyte?

Reply:
the too links here are broken for me in the vercel preview.